design-system-docs/frontend/styles/_footer.scss

```diff
@@ -0,0 +1,37 @@
+    }
+  }
```

Comment:
This is more complex than it needs to be a) because the footer logo is a link, should it be? and b) because we don't provide a standard way of either customising this logo or reversing it out.

Reply:
Personally I don't mind it being a link but I now that I think about it, I have more questions than answers 😅 
- if we use the main CA logo, should it not link to the public website site instead?
- can we insert the svg, similarly to how it's done in the header, and then replace every white fill with currentColour ( this also depends a lot on the logo itself)?
- should the header logo and the footer logo have the same focus state?

Reply:
Yep, I had the same thoughts. I agree making it an SVG would be useful and probably should have the same focus states. I'd be surprised if anyone knew it was a link or clicked on it. We could probably find out from heatmaps of the public site.